wavefront_lambda/wavefront_lambda/__init__.py

```diff
@@ -23,27 +23,27 @@ def call_lambda_with_standard_metrics(wf_reporter, *args, **kwargs):
         # Set cold start counter
         global is_cold_start
         if is_cold_start:
-            aws_cold_starts_counter = delta.delta_counter(reg, "coldstarts")
+            aws_cold_starts_counter = delta.delta_counter(reg, "aws.lambda.wf.coldstarts")
```

Comment:
Could you define the prefix as a constant and just append here. Will make it easier to change in one place if we need to.